app/screens/sso/index.tsx

```diff
@@ -155,14 +156,28 @@ const SSO = ({
         theme,
     };
 
+    let authentication;
+    if (config.MobileExternalBrowser === 'true') {
```

Comment:
@enahum any chance we can or should make it easier to pass an env so that we can be test without having to update the server?

I was thinking about E2E test, if we need to test this.

Reply:
I think it could be done by the time we introduce e2e for this. That e2e will require the server to be configured with a SSO anyway